base/io.jl

```diff
@@ -152,15 +155,35 @@ read(stream, t)
     write(filename::AbstractString, x)
 
 Write the canonical binary representation of a value to the given I/O stream or file.
-Return the number of bytes written into the stream.   See also [`print`](@ref) to
+Return the number of bytes written into the stream. See also [`print`](@ref) to
 write a text representation (with an encoding that may depend upon `io`).
 
+The endianness of the written value depends on the endianness of the host system. Use
+[`hton`](@ref) when writing and [`ntoh`](@ref) when reading to get results that are
+consistent across plattforms.
```

Comment:
Probably better to recommend `htol`, since that is cheaper (free) on most systems.